benchmark/torch/a2c/actor.py

```diff
@@ -67,17 +66,13 @@ def sample(self):
             env_sample_data[env_id] = defaultdict(list)
         for i in range(self.config['sample_batch_steps']):
             self.obs_batch = np.stack(self.obs_batch)
-            self.obs_batch = torch.from_numpy(self.obs_batch).float()
-            if self.actor_cuda:
-                self.obs_batch = self.obs_batch.cuda()
-
-            action_batch, value_batch = self.agent.sample(self.obs_batch)
+            with torch.no_grad():
```

Comment:
Can we remove this line? 
As we have implemented the agent, it can be used to interact with the environment. Pure Python is enough for the interaction. We should remove the logic related to torch.

Reply:
Yes, fixed it.